flybywiresim/a32nx
# fix:  windshield wipers animation

Fixes #2716

## Summary of Changes
Improves the speed and movement of the wipers so that they operate more realistically.
⚠ This will not fix the issue about the wipers not wiping the rain drops away, that is an Asobo issue we can't fix ⚠
## Videos
**All the videos are recorded using the Slow mode on the wipers**
**Before :**

https://user-images.githubusercontent.com/76154362/108213471-59dd2980-712f-11eb-8738-64bb8b720d81.mp4

**After (the recording made my sim stutter a bit) :**

https://user-images.githubusercontent.com/76154362/108213476-5c3f8380-712f-11eb-8ae1-37d8ddd97230.mp4

## References

Original video from #2716 :

https://user-images.githubusercontent.com/76154362/108213500-63ff2800-712f-11eb-8b66-97ced95b91e3.mp4


## Additional context
Discord username (if different from GitHub): Inexorable

## Testing instructions

- Turn the wiper rotary knob to "Slow" and check how the wipers behave
- Turn the wiper rotary knob to "Fast" and check how the wipers behave

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created and uploaded.
The build script will have already been run with the latest changes, so no need to rerun it once you download the zip.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the right side, click on the **Artifacts** drop down and click the **A32NX** link
